specs/AppNotifications/AppNotificationContentSpec/AppNotificationBuilder-spec.md

```diff
@@ -0,0 +1,1170 @@
+
+![Inline Image Example](InlineImageExample.png)
+
+AppNotificationContent code:
+
+```c#
+AppNotificationContent()
+    .AddText(L"Below is an inline image")
+    .SetInlineImage(winrt::Windows::Foundation::Uri(LR"(https://unsplash.it/360/202?image=883)"))
+    .GetXml();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+      <text>Below is an inline image</text>
+      <image src="https://unsplash.it/360/202?image=883">
+    </binding>
+  </visual>
+</toast>
+```
+
+## AppLogoOverride
+
+An image that is left-aligned with notification text.
+
+![AppLogoOverride Image Example](AppLogoOverrideImageExample.png)
+
+AppNotificationContent code:
+
+```c#
+AppNotificationContent()
+    .AddText(L"To the left is an AppLogoOverride")
+    .SetAppLogoOverride(winrt::Windows::Foundation::Uri(LR"(https://unsplash.it/64?image=669)"))
+  .GetXml();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+      <text>To the left is an AppLogoOverride</text>
+      <image src="https://unsplash.it/64?image=669"
+                placement="appLogoOverride" />
+    </binding>
+  </visual>
+</toast>
+```
+
+## Hero
+
+Prominently displays image within the AppNotification banner and while inside Notification Center.
+
+![Hero Image Example](HeroImageExample.png)
+
+AppNotificationContent code:
+
+```c#
+AppNotificationContent()
+    .AddText(Text(L"Above is a Hero image"))
+    .SetHeroImage(winrt::Windows::Foundation::Uri(LR"(https://unsplash.it/360/180?image=1043)"))
+  .GetXml();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+      <text>Above is a Hero image</text>
+      <image src="https://unsplash.it/360/180?image=1043"
+                placement="hero" />
+    </binding>
+  </visual>
+</toast>
+```
+
+## Cropped AppLogoOverride
+
+For both Inline and AppLogoOverride, developers can set how the image will be cropped.
+
+![Cropped AppLogoOverride Image Example](CroppedAppLogoOverrideExample.png)
+
+AppNotificationContent code:
+
+```c#
+AppNotificationContent()
+    .AddText(L"To the left is a cropped AppLogoOverride")
+    .SetAppLogoOverride(winrt::Windows::Foundation::Uri(LR"(https://unsplash.it/64?image=669)"), ImageCrop::Circle)
+    .Show();
+```
+
+XML output:
+
+```c#
+<toast>
+  <visual>
+    <binding template="ToastGeneric">
+      <text>To the left is a cropped AppLogoOverride</text>
+      <image src="https://unsplash.it/64?image=669"
+                hint-crop="circle"
+                placement="appLogoOverride" />
+    </binding>
+  </visual>
+</toast>
+```
+
+# Audio
+
+The Audio component allows the developer to define a custom audio to play when an AppNotification is
+displayed. The audio file can be defined by string value pointing to an app asset or one of the
+[ms-winsoundevents](https://docs.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-audio).
```

Comment:
don't use the en-us part of the url when linking to docs, so the user can see the doc in their preferred language:
```suggestion
[ms-winsoundevents](https://docs.microsoft.com/uwp/schemas/tiles/toastschema/element-audio).
```